pkg/network/events/monitor.go

```diff
@@ -79,6 +79,15 @@ func UnregisterHandler(handler ProcessEventHandler) {
 	m.UnregisterHandler(handler)
 }
 
+// VerifyEventMonitorLoad returns true if init succeeded (used for testing)
```

Comment:
If for testing only, please put in a separate file protected by the `test` build tag